### ISSUE TITLE
Adhere to Angular Package Format (APF) v5.0

### ISSUE THREAD

**Issue by catull** [TwanoO67/ngx-admin-lte#48]
Make sure you fulfil all necessary requirements documented at [APF 5.0](https://goo.gl/jB3GVv).

Came across this error today:

```log
ERROR in ./node_modules/ngx-admin-lte/index.ts
Module build failed: Error: node_modules/ngx-admin-lte/index.ts is missing from the TypeScript compilation. Please make sure it is in your tsconfig via the 'files' or 'include' property.
The missing file seems to be part of a third party library. TS files in published libraries are often a sign of a badly packaged library. Please open an issue in the library repository to alert its author and ask them to package the library using the Angular Package Format (https://goo.gl/jB3GVv).
    at AngularCompilerPlugin.getCompiledFile (node_modules/@ngtools/webpack/src/angular_compiler_plugin.js:674:23)
    at plugin.done.then (node_modules/@ngtools/webpack/src/loader.js:467:39)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:118:7)
```

This is the solution to issues #30 and #46.

**Comment by TwanoO67** [comment]
I just merged your PR and posted a new version "2.0.0-beta-8" on npm.
But it doesn't seems to fix the aot problem.
Did it worked on your version ?

**Comment by catull** [comment]
@TwanoO67 
👍 I could finally, finally eliminate these lines in `tsconfig.json`:

```json
"include": [
  "src/**/*",
  "node_modules/ngx-admin-lte/**/*.ts"
]
```
Thanks for applying the PR.

**Comment by catull** [comment]
This is not implemented in 2.0.0-beta.9, closing it.
